src/main/java/today/sesac/shoutify/post/entity/Post.java

```diff
@@ -0,0 +1,88 @@
+package today.sesac.shoutify.post.entity;
+
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import today.sesac.shoutify.global.domain.BaseEntity;
+
+/**
+ * 게시판의 게시글을 관리하는 도메인입니다.
+ */
+@Entity //엔티티임을 표시
+@Table(name = "posts") //엔티티<->매핑할 테이블 , 이름 생략시 엔티티 이름 사용, 유니크 제약조건
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED) //매개변수 없는 생성자를 protected로 만들어줌
+
+public class Post extends BaseEntity {
+
+    @Id //기본키
+    @GeneratedValue(strategy = GenerationType.IDENTITY) //auto_increment
+    private Long id;
+    /**
+     * 비속어 ai 수정 전 게시물 내용입니다.
+     */
+    @Column(nullable = false, columnDefinition = "TEXT")
+    private String afterContents;
+    /**
+     * 게시물을 작성한 회원입니다
+     */
+//    @ManyToOne
+//    @JoinColumn(name = "author", nullable = false)
+//    private User user;
```

Comment:
P3
이건 지금 고치셔야 하는 건 아닌데, 제가 이전에 이야기 드렸던 건 변수명을 포함해서 말씀드린 거였습니다. ㅎㅎ 나중에 참고해 주세요~!
코드에서도 post.user 같은 형식으로 사용하는 것보다 post.author 로 읽는 게 더 명확하게 이해할 수 있을 것 같아서요!
```suggestion
//    @ManyToOne
//    @JoinColumn(nullable = false)
//    private User author;
```